x-pack/metricbeat/module/prometheus/remote_write/data.go

```diff
@@ -193,17 +191,17 @@ func (g remoteWriteTypedGenerator) GenerateEvents(metrics model.Samples) map[str
 }
 
 // rateCounterUint64 fills a counter value and optionally adds the rate if rate_counters is enabled
-func (g *remoteWriteTypedGenerator) rateCounterUint64(name string, labels mapstr.M, value uint64) mapstr.M {
-	d := mapstr.M{
-		"counter": value,
-	}
-
-	if g.rateCounters {
-		d["rate"], _ = g.counterCache.RateUint64(name+labels.String(), value)
-	}
-
-	return d
-}
+//func (g *remoteWriteTypedGenerator) rateCounterUint64(name string, labels mapstr.M, value uint64) mapstr.M {
+//	d := mapstr.M{
+//		"counter": value,
+//	}
+//
+//	if g.rateCounters {
+//		d["rate"], _ = g.counterCache.RateUint64(name+labels.String(), value)
+//	}
+//
+//	return d
+//}
```

Comment:
Leftover?

Reply:
It's an unused method and hence linter complains about this. I will remove by now.